contribs/gmf/src/services/permalink.js

```diff
@@ -181,6 +269,45 @@ gmf.Permalink.prototype.registerMap_ = function(map) {
         this.ngeoStateManager_.updateState(object);
       }.bind(this), 300, /* invokeApply */ true),
       this);
+
+  // (3) Add map crosshair, if set
+  if (this.getMapCrosshair()) {
+    var crosshairCoordinate;
+    if (center !== null) {
+      crosshairCoordinate = center;
+    } else {
+      crosshairCoordinate = view.getCenter();
+    }
+    goog.asserts.assertArray(crosshairCoordinate);
+
+    var crosshairFeature = new ol.Feature(
+        new ol.geom.Point(crosshairCoordinate))
+    crosshairFeature.setStyle(this.crosshairStyle_);
+    this.featureOverlay_.addFeature(crosshairFeature);
+  }
+
+  // (4) Add map tooltip, if set
+  var tooltipText = this.getMapTooltip();
+  if (tooltipText) {
+    var tooltipPosition;
+    if (center !== null) {
+      tooltipPosition = center;
+    } else {
+      tooltipPosition = view.getCenter();
+    }
+    goog.asserts.assertArray(tooltipPosition);
+
+    var div = $('<div/>')[0];
```

Comment:
It can't.  An `ol.Overlay` takes an `Element`, not a `jQuery` as the `element` property.